### PR TITLE
Remove batched query option

### PR DESCRIPTION
**Pull request by apmoriarty**

### CODE REVIEW

File: warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java [hunk 1]
```diff
@@ -373,7 +366,6 @@ public class QueryOptions implements OptionDescriber {
     protected boolean compressResults = false;
     
     protected Boolean compressedMappings = false;
-    protected boolean limitOverride = false;
```
Comment (jwomeara) [inline warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java:382]: It appears that this variable isn't used anywhere and that's why you removed it.  Just want to make sure I understand this correctly.

**Review by jwomeara — DISMISSED** [review]
Effort to remove batched query looks good.  Can I ask why it's being removed though?  Are we just not using it?  Is it buggy, or non-performant?  Just curious, otherwise looks good to go.

**Review by avgAGB — DISMISSED** [review]

File: warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java (hunk 1 above)
Comment (apmoriarty) [reply warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java:382]: Correct, this is just removing an unused variable.

**Comment by apmoriarty** [comment]
> Effort to remove batched query looks good. Can I ask why it's being removed though? Are we just not using it? Is it buggy, or non-performant? Just curious, otherwise looks good to go.

We evaluated it several years ago and found it to be buggy. It's been disabled via configs as far as I can remember. At one point we considered investing the time to make it work correctly, but decided not to.

**Review by avgAGB — APPROVED** [review]

**Review by jwomeara — APPROVED** [review]

**Pull request merged**